executor/executor.cc

```diff
@@ -3,6 +3,9 @@
 
 // +build
 
+// Currently this is unused (included only to test building).
+#include "flatrpc.h"
```

Comment:
Do you plan to move the relevant functionality completely within executor or was this just a convenient place to include to test building since syz-fuzzer is currently in go? Asking because I was under the impression executor should keep as little state as possible.

Reply:
The first good step is to switch syz-fuzzer to flatrpc, but ultimate plan is to have this in executor.
However, I think we can clearly split it into 2 modes: one is beefy syz-fuzzer mode with more state, etc + simpler slim mode as we have now. The beefy mode will re-exec itself with some env/argument and the child process will detect this in main and do simpler things only.
This has logistical advantage of having only 1 binary. The simpler mode will have more code linked in, but if it's only "passive" code I don't think it matters a lot.

Reply:
Ah this is a good idea.